File: CPE-1star/UVA10041-Vito'sfamily .py
#求中位數，再求中位數距離每個鄰居的和
import statistics
while True:
	try:
		n=int(input())
		for i in range(n):
			nums=list(map(int,input().split()))
			nums.pop(0)
			mid=statistics.median(nums)
			distance=0
			for j in range(len(nums)):
				distance+=abs(mid-nums[j])
			print(int(distance))
	except EOFError:
		break

File: CPE-1star/UVA10041-Vito'sfamily .cpp
#include <iostream>
#include <vector>
#include <algorithm>
using namespace std;

int main() {
    int n;
    while (cin >> n) {
        for (int i = 0; i < n; i++) {
            int m;
            cin >> m;
            vector<int> nums(m);
            for (int j = 0; j < m; j++) {
                cin >> nums[j];
            }
            sort(nums.begin(), nums.end());
            int length = nums.size();
            int mid = nums[length / 2];
            int distance = 0;
            for (int j = 0; j < length; j++) {
                distance += abs(mid - nums[j]);
            }
            cout << distance << "\n";
        }
    }
    return 0;
}
